tools/system_libs.py

```diff
@@ -178,6 +179,21 @@ def create_gl(libname): # libname is ignored, this is just one .o file
     check_call([shared.PYTHON, shared.EMCC, shared.path_from_root('system', 'lib', 'gl.c'), '-o', o])
     return o
 
+  def create_compiler_rt(libname):
+    srcdir = shared.path_from_root('system', 'lib', 'compiler-rt')
+    filenames = ['divdc3.c', 'divsc3.c', 'muldc3.c', 'mulsc3.c']
+    files = (os.path.join(srcdir, f) for f in filenames)
+
+    o_s = []
+    commands = []
+    for src in files:
+      o = in_temp(os.path.basename(src) + '.o')
+      commands.append([shared.PYTHON, shared.EMCC, shared.path_from_root('system', 'lib', src), '-o', o])
```

Comment:
we need an optimization level here, since these should be fast. we use `-Oz` for large code, and `-O2` otherwise, so probably `-O2` is good here.